docs/pages/insights/total-chart.tsx

```diff
@@ -73,17 +87,65 @@ export const TotalChart = memo(
           px="6"
           bg={["whiteAlpha.500", "blackAlpha.300"]}
         >
-          <Center>
-            <Skeleton isLoaded={!isLoading} minW="6ch" rounded="md">
+          <Skeleton isLoaded={!isLoading} rounded="md">
+            <HStack>
               <CountUp
-                fontSize="3xl"
+                minW="2.5ch"
+                fontSize="4xl"
+                lineHeight="1"
                 fontWeight="semibold"
-                count={!isLoading ? score.total : null}
+                count={!isLoading ? currentScore.total : null}
               />
-            </Skeleton>
-          </Center>
+
+              {isNumber(trend) ? (
+                <Tag
+                  colorScheme={
+                    trend === 0 ? "neutral" : trend < 0 ? "danger" : "success"
+                  }
+                >
+                  {trend}%
+                </Tag>
+              ) : null}
+            </HStack>
+          </Skeleton>
 
           <HStack>
+            <Grid
+              display={{ base: "grid", sm: "none" }}
+              templateColumns={{ base: "repeat(4, 1fr)" }}
+              gapX="md"
+              gapY="xs"
+            >
```

Comment:
do you really need the `base: ""` for `display` and `templateColumns`?
```suggestion
            <Grid
              display={{ sm: "none" }}
              templateColumns="repeat(4, 1fr)"
              gapX="md"
              gapY="xs"
            >
```